Token escrow amendment updates/fixes (#3336)
XRPLF/xrpl-dev-portal: docs/references/protocol/transactions/types/accountset.md

|:----------------------------------|:--------------|:--------------|
| `asfAccountTxnID`                 | 5             | Track the ID of this account's most recent transaction. Required for [`AccountTxnID`](../common-fields.md#accounttxnid) |
| `asfAllowTrustLineClawback`       | 16            | Allow account to claw back tokens it has issued. _(Requires the Clawback amendment.)_ Can only be set if the account has an empty owner directory (no trust lines, offers, escrows, payment channels, checks, or signer lists). After you set this flag, it cannot be reverted. The account permanently gains the ability to claw back issued assets on trust lines. |

> | `asfAllowTrustLineLocking`        | 17            | Allow Trust Line tokens issued by this account to be held in escrow. If not set, attempts to create escrows with this account's tokens will fail.  After you set this flag, it cannot be disabled. {% amendment-disclaimer name="TokenEscrow" /%} |

Reviewer: - use "enabled" instead of "set" for flags
- backlink to the escrow concept since someone might not know what it is while reading this reference
- This is just my opinion, but it reads easier to say "tokens issued by this account can't be escrowed" (passive voice, admittedly, but I think an appropriate use since it literally doesn't matter who's trying to do the action) rather than "attempts to create escrows with this account's tokens will fail" (future tense, also less explicit that by "this account's tokens" we mean as an issuer, not as a holder)

```suggestion
| `asfAllowTrustLineLocking`        | 17            | Allow Trust Line tokens issued by this account to be held in [escrow](../../../../concepts/payment-types/escrow.md). If not enabled, tokens issued by this account can't be escrowed. After you enable this flag, it cannot be disabled. {% amendment-disclaimer name="TokenEscrow" /%} |
```
